app/src/main/res/layout/layout_token_info.xml

```diff
@@ -22,20 +22,24 @@
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:orientation="vertical"
+                    android:gravity="center_vertical"
+                    android:orientation="horizontal"
                     android:padding="16dp">
 
                     <TextView
-                        android:layout_width="wrap_content"
+                        android:layout_width="70dp"
                         android:layout_height="wrap_content"
-                        android:text="@string/token_name"
-                        android:textColor="@color/gnosis_blue"
+                        android:text="@string/name"
+                        android:textColor="@color/gnosis_dark_blue_alpha_70"
+                        android:textSize="13sp"
```

Comment:
same as in the other pr ... can we use standard sizes? Didn't you create a style for this in the other PR?